packages/firebase_core/firebase_core_web/lib/src/interop/core.dart

```diff
@@ -20,37 +20,39 @@ List<App> get apps => firebase_interop.apps
     .map((dynamic e) => App.getInstance(e))
     .toList();
 
-App initializeApp(
-    {String apiKey,
-    String authDomain,
-    String databaseURL,
-    String projectId,
-    String storageBucket,
-    String messagingSenderId,
-    String name,
-    String measurementId,
-    String appId}) {
+App initializeApp({
+  String? apiKey,
+  String? authDomain,
+  String? databaseURL,
+  String? projectId,
+  String? storageBucket,
+  String? messagingSenderId,
+  String? name,
+  String? measurementId,
+  String? appId,
+}) {
   name ??= defaultFirebaseAppName;
 
-  try {
-    return App.getInstance(firebase_interop.initializeApp(
-        firebase_interop.FirebaseOptions(
-            apiKey: apiKey,
-            authDomain: authDomain,
-            databaseURL: databaseURL,
-            projectId: projectId,
-            storageBucket: storageBucket,
-            messagingSenderId: messagingSenderId,
-            measurementId: measurementId,
-            appId: appId),
-        name));
-  } catch (e) {
-    // TODO if error - firebase not loaded?
-    rethrow;
-  }
+  return App.getInstance(
+    firebase_interop.initializeApp(
+      firebase_interop.FirebaseOptions(
+        apiKey: apiKey,
+        authDomain: authDomain,
+        databaseURL: databaseURL,
+        projectId: projectId,
+        storageBucket: storageBucket,
+        messagingSenderId: messagingSenderId,
+        measurementId: measurementId,
+        appId: appId,
+      ),
+      name,
+    ),
+  );
+  // TODO if error - firebase not loaded?
```

Comment:
Can this be removed now - we're not checking for an error here anymore so I don't think this TODO is valid anymore

Reply:
I don't think so. My understanding is that this TODO is about adding a try/catch to handle the scenario where firebase wasn't loaded

The try/catch I removed technically did nothing

Reply:
sgtm